backend/prompt_studio/prompt_studio_output_manager/output_manager_helper.py

```diff
@@ -144,3 +145,37 @@ def fetch_default_llm_profile(tool: CustomTool) -> ProfileManager:
             return ProfileManager.get_default_llm_profile(tool=tool)
         except DefaultProfileError:
             raise DefaultProfileError("Default ProfileManager does not exist.")
+
+    @staticmethod
+    def fetch_default_response(
```

Comment:
NIT: @harini-venkataraman naming it as `fetch_default_response` leaves room for doubt and I can't figure out what this function does without checking its code. I suggest renaming it to make it more explicit or adding a docstring describing this